circleci: failed tests: TestVetShadow

The following test appears to have failed:

[#17095](https://circleci.com/gh/cockroachdb/cockroach/17095):

```
=== RUN TestVet
--- PASS: TestVet (0.0s)
=== RUN TestVetShadow
acceptance/cluster/localcluster.go:794: declaration of resp shadows declaration at acceptance/cluster/localcluster.go:788: 
acceptance/cluster/localcluster.go:809: declaration of resp shadows declaration at acceptance/cluster/localcluster.go:788: 
--- FAIL: TestVetShadow (0.0s)
FAIL check-style 0.0s
Makefile:194: recipe for target 'check' failed
make: *** [check] Error 1

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/cluster/localcluster.go
+++ b/acceptance/cluster/localcluster.go
@@ -785,13 +785,13 @@ func (l *LocalCluster) ExecRoot(i int, cmd []string) error {
 			AttachStderr: true,
 			AttachStdout: true,
 		}
-		resp, err := l.client.ContainerExecCreate(ctx, l.Nodes[i].Container.id, cfg)
+		createResp, err := l.client.ContainerExecCreate(ctx, l.Nodes[i].Container.id, cfg)
 		if err != nil {
 			return err
 		}
 		var outputStream, errorStream bytes.Buffer
 		{
-			resp, err := l.client.ContainerExecAttach(ctx, resp.ID, cfg)
+			resp, err := l.client.ContainerExecAttach(ctx, createResp.ID, cfg)
 			if err != nil {
 				return err
 			}
@@ -806,7 +806,7 @@ func (l *LocalCluster) ExecRoot(i int, cmd []string) error {
 			}
 		}
 		{
-			resp, err := l.client.ContainerExecInspect(ctx, resp.ID)
+			resp, err := l.client.ContainerExecInspect(ctx, createResp.ID)
 			if err != nil {
 				return err
 			}

